src/Views/Me/Book.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import DeleteBook from './DeleteBook'
+
+ const Book = ({accessToken, data}) => {
+    const {
+        name,
+        author,
+        publisher,
+        genres,
+        _id
+    } = data;
+
+    const authorList = author.map((singleAuthor, index) => {
```

Comment:
There is a function in shared functions that will make it for you :)

Reply:
I've decided to use own component becasue it was easier to implement for me. If I have some time, I'll try to change it. 

Ok, I will use listProperties

Reply:
"listProperties" function was added to Book component.